### ISSUE TITLE
Custom entry type check on every program launch

### ISSUE THREAD

**Issue by wujastyk** [JabRef/jabref#9930]
### JabRef version

Latest development branch build (please note build date below)

### Operating system

GNU / Linux

### Details on version and operating system

Linux Mint 21.1 with Cinnamon

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

JabRef 5.10--2023-05-20--8d1bb00
Linux 5.15.0-72-generic amd64 
Java 20.0.1 
JavaFX 20+19

1. Start the program
2. a popup window opens "select all customized types to be stored in local preferences"
3. Whether you tick the box or not, this procedure repeats with every program launch.  
4. Similarly, it repeats whether or not you are saving program settings to a USB.

https://github.com/JabRef/jabref/assets/762246/dc94f06a-c0c3-4beb-998f-38b45e0dbe06

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

**Comment by Siedlerchr** [comment]
Hmm. @koppor might this also explain the backup issue?

**Comment by koppor** [comment]
@Siedlerchr I don't think so. Here, custom entry types are written, but during loading not detected that they exist. The Backup Manager does not wrote them at all.

**Comment by Siedlerchr** [comment]
@wujastyk  I am unable to reproduce this in the latest dev version.
Created a new custom entry type, also used it in the library, checked that it's written to the file. e.g.
`@Comment{jabref-entrytype: asdfasdf: req[author] opt[]}`
Closed JabRef. Reopened all fine. Even after deleting the custom entry types from the prefs, the import dialog is only triggered once.

**Comment by wujastyk** [comment]
Hmmm.  It's not happening for me any more either.

JabRef 5.10--2023-05-22--465eea3
Linux 5.15.0-72-generic amd64 
Java 20.0.1 
JavaFX 20+19

**Comment by Siedlerchr** [comment]
Okay, glad to hear! I leave this issue open for a couple of days in case you still encounter it.

**Comment by wujastyk** [comment]
JabRef 5.11--2023-09-13--183375e
Linux 6.2.0-32-generic amd64 
Java 21-internal 
JavaFX 20+19

Still happening.

**Comment by Siedlerchr** [comment]
This is totally odd, must be something with your library. Can you send it to us (mail at developers [at] jabref.org

**Comment by Siedlerchr** [comment]
@wujastyk  Is this still occuring with the latest dev version?

**Comment by wujastyk** [comment]
JabRef 6.0-alpha2.690--2025-09-05--261ae7c
Linux 6.14.0-29-generic amd64 
Java 24.0.2 
JavaFX 24.0.2+1

yes, still happening.

My main library is attached.

[biblio4-utf8.zip](https://github.com/user-attachments/files/22306562/biblio4-utf8.zip)

**Comment by koppor** [comment]
I tried today on CachyOS and latest `main`. It worked as expected here.

@wujastyk Could you show us your preferences after the first opening?

Here, it looks like:

<img width="1132" height="756" alt="Image" src="https://github.com/user-attachments/assets/25042700-b03b-4323-aeea-e6db16d6efdb" />

**Comment by ryan-carpenter** [comment]
@koppor, I have the same results as you with the latest version on openSUSE. Incidentally, there are duplicate keys for `shar-1993`.

JabRef 100.0.0
Linux 6.15.7-1-default amd64 
Java 24.0.1 
JavaFX 24.0.2+1

**Comment by ryan-carpenter** [comment]
> Even after deleting the custom entry types from the prefs, the import dialog is only triggered once

Same here.

**Comment by wujastyk** [comment]
JabRef 6.0-alpha2.721--2025-09-12--4c64b0a
Linux 6.14.0-29-generic amd64 
Java 24.0.2 
JavaFX 24.0.2+1

I have done a video to show the sequence of things.

https://github.com/user-attachments/assets/b5727e9c-5470-4361-8dd6-add48af1af50
